brainzutils/metrics.py

```diff
@@ -74,6 +77,7 @@ def set(metric_name: str, tags: Dict[str, str] = None, timestamp: int = None, **
     metric = "%s,%s %s %d" % (metric_name, tag_string, fields, timestamp)
     try:
         cache._r.rpush(REDIS_METRICS_KEY, metric)
-    except Exception:
+    except Exception as err:
+        logging.error("Cannot set redis metric" % str(err))
         # If we fail to push the metric to redis, so be it.
```

Comment:
Lets remove this comment and the `pass`. I'll fix it.

Reply:
thx